index.js

```diff
@@ -693,46 +745,77 @@ const autoScript = {
     combineAndWriteZones(cb)
   }],
   zipGeoJson: ['mergeZones', function (results, cb) {
-    overallProgress.beginTask('Zipping geojson')
-    exec('zip dist/timezones.geojson.zip dist/combined.json', cb)
+    if (argv.skip_zip) {
+      overallProgress.beginTask('Skipping zip')
+    } else {
+      overallProgress.beginTask('Zipping geojson')
+      let zipFile = argv.dist_dir + '/timezones.geojson.zip'
+      let jsonFile = argv.dist_dir + '/combined.json'
+      exec('zip ' + zipFile + ' ' + jsonFile, cb)
+    }
   }],
   zipGeoJsonWithOceans: ['mergeZones', function (results, cb) {
-    overallProgress.beginTask('Zipping geojson with oceans')
-    exec('zip dist/timezones-with-oceans.geojson.zip dist/combined-with-oceans.json', cb)
+    if (argv.skip_zip) {
+      overallProgress.beginTask('Skipping with oceans zip')
+    } else {
+      overallProgress.beginTask('Zipping geojson with oceans')
```

Comment:
The callback `cb` needs to be called in this block in order to indicate that this task has completed successfully. Also, one small code style guide is that it could be cleaner to include a return statement in this block and then the rest of the method won't be so indented as much. ie:

```js
if (argv.skip_zip) {
  overallProgress.beginTask('Skipping with oceans zip')
  return cb()
}
overallProgress.beginTask('Zipping geojson with oceans')
...
```

Reply:
Done in commit 5